packages/docusaurus-theme-common/src/utils/ThemeClassnames.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export const ThemeClassNames = {
```

Comment:
I mean add a little // comment here so that someone reading the code understands why we created this file in the first place. Nothing too complicated, just 1 line